src/GAP.jl

```diff
@@ -96,6 +96,13 @@ function initialize(argv::Vector{String})
     # TODO: turn this into a proper libgap API
     unsafe_store!(cglobal((:SyLoadSystemInitFile, libgap), Int64), 0)
 
+    # remember sigchld handler
+    if Sys.isbsd() || Sys.islinux()
+        sigchld = Sys.isbsd() ? 20 : 17
+        # nullptr == SIG_DFL
+        prev_sigchld_hdl = ccall(:signal, Ptr{Cvoid}, (Cint, Ptr{Cvoid}), sigchld, Ptr{Cvoid}(0))
```

Comment:
Usually it will be `NULL`, but there might be other software as well that sets a sigchld handler so to be safe I wanted to reinstate whatever was set before GAP replaces it instead of the default.

Reply:
Oh sure, I agree this is the right way to do it. I was just wondering